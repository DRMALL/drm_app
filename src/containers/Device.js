import React, { Component } from 'react'
import { View, Text, Image } from 'react-native'

const deviceIconSelected = require('../images/tabbar_icons/tabbar_home_selected.png')
    , homeIconNormal = require('../images/tabbar_icons/tabbar_home_normal.png')

export default class Device extends Component {
  static navigationOptions = {
    tabBarIcon: ({ tintColor }) => (
      <Image
        source={deviceIconSelected}
        style={{tintColor: tintColor}}
      />
    ),
  }
  render() {
    return(
      <View>
        <Text>Hi I Am Device</Text>
      </View>
    )
  }
}