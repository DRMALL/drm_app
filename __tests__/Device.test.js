import 'react-native'
import React from 'react'
import Device from '../src/containers/Device'
import renderer from 'react-test-renderer';

test('Device test', () => {
  const tree = renderer.create(
    <Device />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})