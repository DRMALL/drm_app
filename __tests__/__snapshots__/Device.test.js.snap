// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Device test 1`] = `
<View
  style={
    Object {
      "paddingBottom": 100,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#F7F7F7",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderBottomWidth": 0.5,
          "borderColor": "rgba(0,0,0,.06)",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-around",
          "width": "100%",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "rgba(0,0,0,.06)",
            "borderLeftWidth": 0.5,
            "borderRightWidth": 0.5,
            "flexDirection": "row",
            "justifyContent": "center",
            "opacity": 1,
            "paddingVertical": 10,
            "width": "33.333%",
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "fontSize": 15,
                "marginRight": 5,
              },
              Object {
                "color": "rgba(0,0,0,.6)",
              },
            ]
          }
        >
          分类
        </Text>
        <Image
          source={1}
        />
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "rgba(0,0,0,.06)",
            "borderLeftWidth": 0.5,
            "borderRightWidth": 0.5,
            "flexDirection": "row",
            "justifyContent": "center",
            "opacity": 1,
            "paddingVertical": 10,
            "width": "33.333%",
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "fontSize": 15,
                "marginRight": 5,
              },
              Object {
                "color": "rgba(0,0,0,.6)",
              },
            ]
          }
        >
          排序
        </Text>
        <Image
          source={1}
        />
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "rgba(0,0,0,.06)",
            "borderLeftWidth": 0.5,
            "borderRightWidth": 0.5,
            "flexDirection": "row",
            "justifyContent": "center",
            "opacity": 1,
            "paddingVertical": 10,
            "width": "33.333%",
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "fontSize": 15,
                "marginRight": 5,
              },
              Object {
                "color": "rgba(0,0,0,.6)",
              },
            ]
          }
        >
          筛选
        </Text>
        <Image
          source={1}
        />
      </View>
    </View>
    <View
      style={
        Object {
          "height": "100%",
          "paddingBottom": 50,
        }
      }
    >
      <View
        style={
          Object {
            "position": "relative",
            "zIndex": 3,
          }
        }
      >
        <RCTScrollView
          dataSource={
            ListViewDataSource {
              "items": 0,
            }
          }
          enableEmptySections={true}
          refreshControl={
            <RefreshControlMock
              colors={
                Array [
                  "#ff0000",
                  "#00ff00",
                  "#0000ff",
                ]
              }
              onRefresh={[Function]}
              progressBackgroundColor="#fff"
              refreshing={false}
              title="下拉刷新"
              titleColor="rgba(0,0,0,.6)"
            />
          }
          renderRow={[Function]}
          renderScrollComponent={[Function]}
          style={
            Object {
              "height": "100%",
            }
          }
        >
          <RCTRefreshControl />
          <View />
        </RCTScrollView>
      </View>
      <View
        style={
          Array [
            Object {
              "position": "relative",
              "zIndex": 3,
            },
            Object {
              "height": 0,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "height": 0,
            }
          }
        />
      </View>
      <View
        style={
          Array [
            Object {
              "position": "relative",
              "zIndex": 3,
            },
            Object {
              "height": 0,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "height": 0,
            }
          }
        />
      </View>
      <View
        style={
          Array [
            Object {
              "position": "relative",
              "zIndex": 3,
            },
            Object {
              "height": 0,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "height": 0,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
